server/portal/apps/projects/workspace_operations/shared_worspace_migration.py

```diff
@@ -0,0 +1,58 @@
+"""
+Migration scripts for bringing Shared Workspaces from V2 to V3.
+"""
+import requests
+from django.conf import settings
+from portal.apps.projects.models import ProjectMetadata
+from portal.apps.projects.workspace_operations.shared_workspace_operations import create_workspace_system, add_user_to_workspace
+from portal.libs.agave.utils import service_account
+
+from portal.settings.settings_secret import _AGAVE_SUPER_TOKEN as v2_token
+from portal.settings.settings_secret import _AGAVE_TENANT_BASEURL as v2_url
+
+ROLE_MAP = {
+    "USER": "writer",
+    "ADMIN": "writer",
+    "GUEST": "reader",
+    "OWNER": None,
```

Comment:
Owner's don't need a role?

Reply:
System owners in Tapis V3 already have the highest possible level of permissions. Really there shouldn't be any case where we have to look up"OWNER" in this map, since it's just used to transfer co-PIs and team members.

Reply:
Does it come into play when determining ACLs? I had thought I saw that the `None` role was read-only ACLs